src/Common/src/System/Net/Http/HttpHandlerDefaults.cs

```diff
@@ -11,5 +11,7 @@ internal static class HttpHandlerDefaults
         public const int DefaultMaxAutomaticRedirections = 50;
         public const DecompressionMethods DefaultAutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate;
         public const bool DefaultAutomaticRedirection = true;
+        public const bool DefaultUseCookies = true;
```

Comment:
@kapilash, if you're referring to the cookie container coming back as null rather than as a default instance, I addressed that in the CookieContainer property by lazily initializing the collection.